smoke-tests/src/test/groovy/io/opentelemetry/smoketest/QuarkusSmokeTest.groovy

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.smoketest
+
+import io.opentelemetry.proto.collector.trace.v1.ExportTraceServiceRequest
+import spock.lang.Ignore
+import spock.lang.IgnoreIf
+import spock.lang.Unroll
+
+import java.util.jar.Attributes
+import java.util.jar.JarFile
+
+import static java.util.stream.Collectors.toSet
+
+// TODO (trask) remove global @Ignore after smoke test image is published
+@Ignore
+@IgnoreIf({ os.windows })
+class QuarkusSmokeTest extends SmokeTest {
+
+  protected String getTargetImage(String jdk) {
+    // TODO (trask) update tag after smoke test image is published
+    "ghcr.io/open-telemetry/opentelemetry-java-instrumentation/smoke-test-quarkus:jdk$jdk-20211212.1701473"
+  }
+
+  @Unroll
+  def "quarkus smoke test on JDK #jdk"(int jdk) {
+    setup:
+    startTarget(jdk)
+
+    def currentAgentVersion = new JarFile(agentPath).getManifest().getMainAttributes().get(Attributes.Name.IMPLEMENTATION_VERSION)
+
+    when:
+    client().get("/hello").aggregate().join()
+    Collection<ExportTraceServiceRequest> traces = waitForTraces()
+
+    then:
+    countSpansByName(traces, '/hello') == 1
```

Comment:
without the fix, the SERVER span name is `/` b/c the correlation is lost between the SERVER span and the JAX-RS routing instrumentation, so the SERVER span name cannot be updated